### ISSUE TITLE
docker import too slow

### ISSUE THREAD

**Issue by paigeadelethompson** [cockroachdb/cockroach#132544]
```
docker run -it --rm --net host -v $(pwd)/dump/:/dump -v $(pwd)/crdb/data:/cockroach/cockroach-data -v $(pwd)/crdb/certs:/certs docker.io/cockroachdb/cockroach:latest sql --certs-dir /certs -f /dump/dump.sql --echo-sql
```

does fuck all though, 

```
COPY 0

Time: 3ms

COPY 121619

Time: 5.785s <---- 

COPY 0

Time: 29ms

COPY 0

Time: 15ms

COPY 0

Time: 6ms

COPY 983

Time: 10.116s

COPY 1680

Time: 1.202s

```
yeah no that didn't take 5 seconds, more like 5 hours. 

```
de94d0e85e1d   cockroachdb/cockroach:latest   "/cockroach/cockroac…"   9 hours ago    Up 9 hours              vibrant_murdock

```

pretty ridiculous this isn't useful to me I can't imagine who this would be useful to, probably nobody 

```
# ls -lah dump/dump.sql 
-rw-r--r-- 1 root root 1.1G Oct 12 12:40 dump/dump.sql
# 
```

Jira issue: CRDB-43133

**Comment by paigeadelethompson** [comment]
It would appear this is some kind of problem with `cockroach sql` 

<img width="1521" alt="image" src="https://github.com/user-attachments/assets/75ebfaf3-26e0-4ff7-b5d6-0af8961452a3">

nothing about this appears to be broken other than sql latency threshold seems to think each query is a little high, this doesn't really seem like a latency issue as much as the client is just sitting there not doing anything 

<img width="1521" alt="image" src="https://github.com/user-attachments/assets/9b177b23-d725-4fca-91e5-36baa8a19e67">

**Comment by paigeadelethompson** [comment]
TLDR the client isn't working right, its job is to literally read the statements from the .sql file and execute the statements and it's for all I know doing nothing: (pid 117102) 

<img width="980" alt="image" src="https://github.com/user-attachments/assets/2b14e960-7487-4357-97fd-1678b02c65dd">

```
> ALTER TABLE ONLY public.userapi_pushers ALTER COLUMN id SET DEFAULT nextval('public.userapi_pushers_id_seq'::regclass);
ALTER TABLE

Time: 124ms

COPY 22

Time: 21ms

COPY 4698

Time: 313ms

COPY 4204

Time: 287ms

COPY 0

Time: 1ms

```

running for 29 minutes now file size approximately 1GB

**Comment by paigeadelethompson** [comment]
I'm seeing somewhat better results with `pg_restore`, I'll let it run and see what it does

**Comment by paigeadelethompson** [comment]
definitely something wrong with this database too: 

```
pg_restore: processing item 252 TABLE userapi_registration_tokens
pg_restore: creating TABLE "public.userapi_registration_tokens"
pg_restore: processing item 263 TABLE userapi_threepids
pg_restore: creating TABLE "public.userapi_threepids"
pg_restore: processing item 4508 DEFAULT federationsender_queue_json json_nid
pg_restore: creating DEFAULT "public.federationsender_queue_json json_nid"
pg_restore: processing item 4529 DEFAULT syncapi_filter id
pg_restore: creating DEFAULT "public.syncapi_filter id"
pg_restore: processing item 4531 DEFAULT syncapi_notification_data id
pg_restore: creating DEFAULT "public.syncapi_notification_data id"
pg_restore: processing item 4517 DEFAULT userapi_notifications id
pg_restore: creating DEFAULT "public.userapi_notifications id"
pg_restore: processing item 4515 DEFAULT userapi_pushers id
pg_restore: creating DEFAULT "public.userapi_pushers id"
pg_restore: entering main parallel loop
pg_restore: launching item 4878 TABLE DATA roomserver_event_json
pg_restore: launching item 4940 TABLE DATA syncapi_output_room_events
pg_restore: launching item 4941 TABLE DATA syncapi_current_room_state
pg_restore: processing data for table "public.syncapi_output_room_events"
pg_restore: processing data for table "public.syncapi_current_room_state"
pg_restore: processing data for table "public.roomserver_event_json"
pg_restore: while PROCESSING TOC:
pg_restore: from TOC entry 4940; 0 16873 TABLE DATA syncapi_output_room_events avnadmin
pg_restore: error: COPY failed for table "syncapi_output_room_events": ERROR:  expected 13 values, got 1
pg_restore: finished item 4940 TABLE DATA syncapi_output_room_events
pg_restore: launching item 4672 CONSTRAINT syncapi_output_room_events syncapi_output_room_event_id_idx
pg_restore: creating CONSTRAINT "public.syncapi_output_room_events syncapi_output_room_event_id_idx"
pg_restore: while PROCESSING TOC:
pg_restore: from TOC entry 4941; 0 16892 TABLE DATA syncapi_current_room_state avnadmin
pg_restore: error: COPY failed for table "syncapi_current_room_state": ERROR:  expected 10 values, got 1
pg_restore: finished item 4941 TABLE DATA syncapi_current_room_state
pg_restore: launching item 4682 INDEX syncapi_current_room_state_eventid_idx
pg_restore: creating INDEX "public.syncapi_current_room_state_eventid_idx"
pg_restore: finished item 4682 INDEX syncapi_current_room_state_eventid_idx
pg_restore: launching item 4683 INDEX syncapi_current_room_state_type_state_key_idx
pg_restore: creating INDEX "public.syncapi_current_room_state_type_state_key_idx"
pg_restore: finished item 4683 INDEX syncapi_current_room_state_type_state_key_idx
pg_restore: launching item 4684 INDEX syncapi_event_id_idx
pg_restore: creating INDEX "public.syncapi_event_id_idx"
pg_restore: finished item 4684 INDEX syncapi_event_id_idx
pg_restore: launching item 4685 INDEX syncapi_membership_idx
pg_restore: creating INDEX "public.syncapi_membership_idx"
pg_restore: finished item 4685 INDEX syncapi_membership_idx
pg_restore: launching item 4687 CONSTRAINT syncapi_current_room_state syncapi_room_state_unique
pg_restore: creating CONSTRAINT "public.syncapi_current_room_state syncapi_room_state_unique"
pg_restore: finished item 4687 CONSTRAINT syncapi_current_room_state syncapi_room_state_unique
pg_restore: launching item 4880 TABLE DATA roomserver_events
pg_restore: processing data for table "public.roomserver_events"
pg_restore: from TOC entry 4880; 0 16459 TABLE DATA roomserver_events avnadmin
pg_restore: error: COPY failed for table "roomserver_events": ERROR:  expected 10 values, got 1
pg_restore: finished item 4880 TABLE DATA roomserver_events
pg_restore: launching item 4546 INDEX roomserver_event_event_type_nid_idx
pg_restore: creating INDEX "public.roomserver_event_event_type_nid_idx"
```

<img width="1061" alt="image" src="https://github.com/user-attachments/assets/05114946-f8e6-48c1-961e-3854c0ca89a3">

but its characterized by these long periods of doing seemingly nothing and then all of a sudden starts doing something. Working with `pg_dump` and `pg_restore` for PG16
